src/cms/adobe-stock-manage.md

```diff
@@ -0,0 +1,210 @@
+
+    ![Adobe Stock search results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
+    _Adobe Stock search results_
+
+## Filter and Search for Adobe Stock Assets
+
+The [Adobe Stock Search Grid][access-search] provides querying and filtering functionality to help you find the perfect image for your Magento stores.
+
+By default, the search results shown are from an Adobe Stock curated gallery of a few hundred results. As soon as you apply your own keyword search, you will be
+searching the millions of assets available via Adobe Stock.
+
+### Search for Adobe Stock Assets by Keywords
+
+1. [Access the Adobe Stock Search grid][access-search].
+
+1. Enter your keyword search into the **Search by keyword** input field in the top-left and click the magnifying glass or press **Enter**.
+
+    ![Adobe Stock Search Results for the "mango" keyword]({% link images/images/adobe-stock/mango.png %}){: .zoom}
+    _Adobe Stock search results - "mango" keyword_
+
+### Filter Adobe Stock Assets
+
+1. [Run a keyword search for Adobe Stock assets][search-by-keywords].
+
+1. Click <span class="btn">Filters</span>.
+
+    There are several filters available to refine your search results:
+
+    |Filter|Description|
+    |---|---|
+    |Subcategory|Filter for images that are **Photos** or **Illustrations**|
+    |Orientation|Filter for images by size, shape and aspect|
+    |Color|Use a color pallette to filter for images by color|
+    |Price|Filter for images based on their cost|
+    |Safe search|Enable or disable Safe search|
+    |Isolated Assets|Show only Isolated Assets, which have subjects appear alone on a solid background|
+
+    ![Adobe Stock search filters]({% link images/images/adobe-stock/filters.png %}){: .zoom}
+    _Adobe Stock Search Filters_
+
+1. Click **Apply Filters**.
+
+    The search result grid is updated with your refined search.
+
+## View Image Details
+
+Each image has details available for viewing. Additional image-specific actions, such as [saving image previews][save-preview] or [saving (and optionally licensing) images][save-licensed], are available via this detailed view.
+
+1. [Access the Adobe Stock Search grid][access-search].
+
+1. Click an image in the search results.
+
+    Further image details are displayed, such as:
+    - A larger version of the image
+    - Image metadata, such as **Dimensions**, **File type**, **Category**, **File#**, and **Keywords**
+    - Related images, such as images from the same **series** or **model**
+    - Action buttons, such as [**Save Preview**][save-preview] and [**Save (and optionally license) Image**][save-licensed]
+
+    ![Adobe Stock image details]({% link images/images/adobe-stock/image-details.png %}){: .zoom}
+    _Adobe Stock image details_
+
+## Log In to Your Adobe Account
+
+To gain complete access to an image and eliminate the Adobe Stock watermark, you must [sign in with an Adobe account][adobe-signin] and purchase credits to
+license rights to use an image.
```

Comment:
is there a soft return here?

Reply:
doesn't affect output, but we will fix later